module/VuFind/src/VuFind/Db/Entity/ChangeTrackerEntityInterface.php

```diff
@@ -40,4 +42,99 @@
  */
 interface ChangeTrackerEntityInterface extends EntityInterface
 {
+    /**
+     * Id setter
+     *
+     * @param string $id Id
+     *
+     * @return ChangeTrackerEntityInterface
+     */
+    public function setId(string $id): ChangeTrackerEntityInterface;
+
+    /**
+     * Id getter
+     *
+     * @return string
+     */
+    public function getId(): string;
+
+    /**
+     * Core setter
```

Comment:
While we're here, would it be feasible to update the terminology? I'd prefer something like "index id" or "index name" instead of "core".

Reply:
How far do you want to take this? It certainly makes sense at a minimum to clarify the comments... but do we also want to change the method names? If we change the method names, it feels like we would also probably want to change the column names in the database... and that probably is farther than we want to go right now.

Reply:
I wouldn't have a problem with keeping the column names while changing the method names, but I can live with clarified comments.

Reply:
The methods aren't actually used yet, so renaming them is harmless enough. :-)